src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball();
```

Comment:
Use color supplier + random here and create a new ball. 
```suggestion
        return new Ball();
```